src/romi-robot.ts

```diff
@@ -73,23 +74,26 @@ export default class WPILibWSRomiRobot extends WPILibWSRobotBase {
     private _readyP: Promise<void>;
     private _i2cErrorDetector: I2CErrorDetector = new I2CErrorDetector(10, 500, 100);
 
+    private _lsm6: LSM6;
+
     // Take in the abstract bus, since this will allow us to
     // write unit tests more easily
     constructor(bus: I2CPromisifiedBus, address: number, ioConfig?: IPinConfiguration[]) {
         super();
 
-        // TODO We should wrap HW initialization in a promise
-        // that readyP() can return
         this._i2cBus = bus;
         this._i2cAddress = address;
 
+        // Set up the LSM6DS33
+        this._lsm6 = new LSM6(this._i2cBus, 0x6B);
```

Comment:
Where does 0x6B come from? Should we move this to a const?

Reply:
It's the hardcoded address of the sensor